source/manual/setting-up-new-rails-app.html.md

```diff
@@ -16,137 +16,174 @@ parent: "/manual.html"
 [release]: https://release.publishing.service.gov.uk/applications
 [deploy-jenkins]: https://deploy.integration.publishing.service.gov.uk/job/Deploy_App/
 [docs-applications]: https://github.com/alphagov/govuk-developer-docs/blob/master/data/applications.yml
+[get-started]: https://docs.publishing.service.gov.uk/manual/get-started.html
+[linting]: https://docs.publishing.service.gov.uk/manual/configure-linting.html
+[rails-conv]: https://docs.publishing.service.gov.uk/manual/conventions-for-rails-applications.html
+[naming]: https://docs.publishing.service.gov.uk/manual/naming.html
+[auto-config]: https://docs.publishing.service.gov.uk/manual/configure-github-repo.html#auto-configuration
+[app-list]: https://docs.publishing.service.gov.uk/#applications
 
-## Familiarise yourself with the conventions
+## Before you start
 
-We have documented [conventions for Rails
-applications](/manual/conventions-for-rails-applications.html) and
-[conventions for naming applications](/manual/naming.html) in GOV.UK.
+Before you create a Rails app, you must complete all steps in the [GOV.UK developer get started documentation](get-started).
```

Comment:
In all the reference links, the syntax is `[title][reference]` - so this should be `[GOV.UK developer get started documentation][get-started]`, with similar changes below.